experiments/AMIP/components/ocean/slab_ocean_init.jl

```diff
@@ -36,30 +36,33 @@ struct OceanSlabParameters{FT <: AbstractFloat}
 end
 name(::SlabOceanSimulation) = "SlabOceanSimulation"
 
-# init simulation
-function slab_ocean_space_init(::Type{FT}, space, p) where {FT}
+function get_T_sfc(params, coord)
+    T_sfc_0 = FT(params.T_init) #- FT(275) # close to the average of T_1 in atmos
+    anomaly = false
+    anomaly_tropics = true
+    anom = FT(0)
+    radlat = FT(coord.lat / 180 * pi)
+    if anomaly == true
+        lat_0 = FT(60 / 180 * pi)
+        lon_0 = FT(-90 / 180 * pi)
+        radlon = FT(coord.long / 180 * pi)
+        stdev = FT(5 / 180 * pi)
+        anom_ampl = FT(0)
+        anom = FT(anom_ampl * exp(-((radlat - lat_0)^2 / 2stdev^2 + (radlon - lon_0)^2 / 2stdev^2)))
+    elseif anomaly_tropics == true
+        anom = FT(29 * exp(-coord.lat^2 / (2 * 26^2)))
+    end
+    T_sfc = FT(T_sfc_0 + anom)
+    return T_sfc
+end
```

Comment:
Lot's of values are hardcoded here, including some booleans. There seem to be some overlap in what is happening here with a previous comment of mine.

```julia
if anomaly == true
```
is equivalent to
```julia
if anomaly
```
when anomaly is a `Bool`.

But `anomaly` is hardcoded to be false. This function should be fixed to do what is supposed and only what it should without hardcoded booleans.

Reply:
@LenkaNovak is it okay to remove the `anomaly == true` branch?